nemo_text_processing/inverse_text_normalization/hi/data/time/hour.tsv

```diff
@@ -0,0 +1,26 @@
+१	एक
```

Comment:
are these different than the numbers defined in the number data dir?

Reply:
Yes, this is different because the digit.tsv file inside the numbers directory contains numbers only up to 9. The teens_and_ties.tsv file covers numbers from 10 to 99. For hour.tsv, I wanted a single file that covers numbers from 1 to 24, to simplify things rather than using the numbers separately from digit.stv and teens_and_ties.tsv which is in the number data dir.

Reply:
I just checked English and English also had a seperate file for hour and minute.

Reply:
please check how minutes are defined in the tagger without a data file here: https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/inverse_text_normalization/en/taggers/time.py#L70